crates/services/gas_price_service/src/v1/da_source_service.rs

```diff
@@ -178,10 +184,11 @@ mod tests {
 
         // given
         let l2_height = 10;
-        let recorded_height = 9;
+        let starting_recorded_height = 2;
```

Comment:
Shouldn't the `starting_recorded_height` be 2 - 1? We call `succ` on the `recorded_height` to query the DA heights.

Reply:
addressed in 53a96cfc31